splink/linker.py

```diff
@@ -1919,3 +1924,50 @@ def load_settings_from_json(self, in_path: str):
         with open(in_path, "r") as f:
             model_dict = json.load(f)
         self.initialise_settings(model_dict)
+
+    def estimate_probability_two_random_records_match(
+        self, deterministic_matching_rules, recall
+    ):
+        """Estimate the model parameter `probability_two_random_records_match` using
+        a direct estimation approach.
+
+        See [here](https://github.com/moj-analytical-services/splink/issues/462)
+        for discussion of methodology
+
+        Args:
+            deterministic_matching_rules (list): A list of deterministic matching
+                rules that should be designed to admit very few (none if possible)
+                false positives
+            recall (float): A guess at the recall the deterministic matching rules
+                will attain.  i.e. what proportion of true matches will be recovered
+                by these deterministic rules
+        """
+
+        # If user, by error, provides a single rule as a string
+        if isinstance(deterministic_matching_rules, str):
```

Comment:
You could also just use [`ensure_is_list`](https://github.com/moj-analytical-services/splink/blob/master/splink/misc.py#L44) here